README.md

```diff
@@ -33,7 +33,7 @@ hello.process(Map.of("name", "bob"), stringWriter);
 String result = stringWriter.toString();
 ```
 
-For more details, check the complete [documentation](doc/freemarker.adoc).
+For more details, check the complete [documentation](docs/modules/ROOT/pages/index.adoc).
```

Comment:
Is it really necessary to put the doc so deep in a hierarchy? Doesn't look very developer friendly?

Reply:
Yes, this is the Antora's Standard File and Directory layout.

See https://docs.antora.org/antora/2.3/standard-directories/#hierarchy-and-reserved-names

Reply:
Actually this link is wrong, let me fix it